src/amo/reducers/users.js

```diff
@@ -141,15 +142,15 @@ type EditUserAccountAction = {|
 |};
 
 export const editUserAccount = ({
-  errorHandlerId, userFields, userId,
+  errorHandlerId, picture, userFields, userId,
 }: EditUserAccountParams): EditUserAccountAction => {
   invariant(errorHandlerId, 'errorHandlerId is required');
   invariant(userFields, 'userFields are required');
   invariant(userId, 'userId is required');
 
   return {
     type: EDIT_USER_ACCOUNT,
-    payload: { errorHandlerId, userFields, userId },
+    payload: { errorHandlerId, userFields, picture, userId },
```

Comment:
Nit: I think we try to keep arguments in alphabetical order.

Reply:
Good catch!